### ISSUE TITLE
[a11y]: Progress indicator component steps "span" and "button" target elements do not have minimum spacing requirements

### ISSUE THREAD

**Issue by anish-mm-05** [carbon-design-system/carbon#18037]
### Package

@carbon/ibm-products

### Browser

Chrome

### Operating System

MacOS

### Package version

v2.18.1

### React version

v18.2.0

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker

### Description

CreateFullPage component steps has span and button target elements so it should meet a minimum spacing requirement of 12px.

![Image](https://github.com/user-attachments/assets/d51c4f82-c5c3-400c-a39f-e6d2a6e5e0c7)

### Reproduction/example

https://stackblitz.com/github/carbon-design-system/ibm-products/tree/main/examples/carbon-for-ibm-products/CreateFullPage?file=src%2FExample%2FExample.jsx

### Steps to reproduce

1. Open the example link or (https://wmaibzpqbagithub-qeho--5173--34c588ed.local-credentialless.webcontainer.io/)
2. Run the IBM Equal Access Accessibility Checker
3. Its shows an issue like this _Undersized target "span" does not have sufficient spacing of 12 CSS pixels from another target "button"_

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/ibm-products/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/ibm-products/issues) for duplicate problems

**Comment by annawen1** [comment]
@amal-k-joy we're not able to replicate the issue, perhaps it's already been resolved? Can you also confirm on your end if we can close this issue?

**Comment by amal-k-joy** [comment]
@annawen1 ,
Looks like this is no longer an issue. I think `accessibility-checker` got that fixed from `06 November 2024 Deployment (IBM 7.3)`

**Comment by heloiselui** [comment]
Thanks for check that @annawen1 e @amal-k-joy, we'll close this issue then.
